### PR TITLE
Basic functionality

### PR DESCRIPTION
**Pull request by ianuragsharma**
Please Check the basic functionality commit you can ignore the initial commit.
https://user-images.githubusercontent.com/41025915/167278297-285e5e27-e46b-4245-841c-56ddd15f83a8.mp4

### CODE REVIEW

**Review by hrishikeshmane — APPROVED** [review]
looks good to me. OK to merge.

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const HomePage = () => {
+  const usernameHandler = (event) => {
+    if (event.key === "Enter") {
+      localStorage.setItem("userName", event.target.value);
+      window.location.reload(false);
```
Comment (shraddha15-creator) [inline src/pages/HomePage.jsx:7]: can't we use ` event.preventDefault() ` here?

**Review by shraddha15-creator — APPROVED** [review]
Code looks good and readable.

File: src/pages/HomePage.jsx (hunk 1 above)
Comment (ianuragsharma) [reply src/pages/HomePage.jsx:7]: no, that works with form only ig.

**Review by pradipgharge — APPROVED** [review]
Looks good to merge to me

**Pull request merged**